doc-content/enterprise-only/getting-started-kogito/con-git-build-operator-deployment-option.adoc

```diff
@@ -0,0 +1,29 @@
+[id="con-git-build-operator-deployment-option_{context}"]
+= Git source build and deployment
+In most use cases, you can use the standard runtime build and deployment method to deploy {KOGITO} microservices on OpenShift from a Git repository source, as shown in the following examples. These examples are based on the `kogito-travel-agency` extended example application.
```

Comment:
`These examples are based on the kogito-travel-agency extended example application.`
We won't support such complex deployment for 7.11 or 7.12. Would be better to use some other example, possibly `dmn-quarkus-example`?